silverback/cluster/types.py

```diff
@@ -311,6 +311,14 @@ def cluster(self) -> ServiceHealth:
         return ServiceHealth(healthy=self.ars.healthy and self.ccs.healthy)
 
 
+class DockerCredentialsInfo(BaseModel):
```

Comment:
Suggestion: Would this be a clearer name to fit what it's purpose is?
```suggestion
class RegistryCredentialsInfo(BaseModel):
```

Reply:
c641cde